envi/archs/i386/emu.py

```diff
@@ -608,10 +651,15 @@ def i_bswap(self, op):
 
     def i_bsr(self, op):
         val = self.getOperValue(op, 1)
-
+        self.setFlag(EFLAGS_AF, False)  # undocumented, but 32-bit VBox on i9 says so
```

Comment:
Integer now boolean

Reply:
ok, and i went through and updated the other 15 lines that use boolean instead of integer to set flags.
but i don't like that you're making a big deal about it in this PR... it's literally a bit, it's 1 or 0, True or False.
and these are mostly Visi's doing, i'm just following the pattern that's there.